scripts/license.ts

```diff
@@ -105,19 +112,29 @@ ${bundledDeps.map((l) => l.content).join('\n')}
   fs.writeFileSync(join(opts.buildDir, 'license-source.txt'), licenseSource.join('\n'));
 }
 
-function createBundledDeps(opts: BuildOptions, bundledDeps: BundledDep[], deps: string[]) {
-  if (Array.isArray(deps)) {
```

Comment:
I removed this `if` statement, as the typings _should_ prevent a non-array from being passed in